bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -262,7 +263,7 @@ protected void doGet(@NotNull SlingHttpServletRequest request, @NotNull SlingHtt
                 metrics.markImageStreamed();
             }
         } catch (IllegalArgumentException e) {
-            LOGGER.error("Invalid image request", e.getMessage());
+            LOGGER.error("Invalid image request" + e.getMessage());
```

Comment:
It would be better to log the exception too: `LOGGER.error("Invalid image request", e)`